ingestion/functions/parsing/riograndedosul/riograndedosul.py

```diff
@@ -0,0 +1,246 @@
+            {
+                "start": convert_date(date_symptoms),
+                "end": convert_date(date_symptoms),
+            },
+        })
+    if hospitalized != "NAO":
+        events.append({
+            "name": "hospitalAdmission",
+            "dateRange":
+            {
+                "start": convert_date(date_confirmed),
+                "end": convert_date(date_confirmed),
+            },
+            "value": "Yes"
+        })
+    if date_death:
+        events.append({
+            "name": "outcome",
+            "dateRange":
+            {
+                "start": convert_date(date_death),
+                "end": convert_date(date_death),
+            },
+            "value": "Death"
+        })
+    return events
+
+
+def convert_symptoms(fever: str, cough: str, sorethroat: str, shortbreath: str, other: str):
+    symptoms = []
+    if fever == "SIM":
+        symptoms.append("Fever")
+    if cough == "SIM":
+        symptoms.append("Cough")
+    if sorethroat == "SIM":
+        symptoms.append("Sore Throat")
+    if shortbreath == "SIM":
+        symptoms.append("Shortness of Breath")
+    if other == "SIM":
+        symptoms.append("Other")
+    return symptoms
+
+
+def convert_preexisting_conditions(raw_commorbidities: str):
+    preexistingConditions = {}
+    if not raw_commorbidities:
+        return None
+    preexistingConditions["hasPreexistingConditions"] = True
+    commorbidities_list = []
+
+    for key in _COMORBIDITIES_MAP:
+        if key in raw_commorbidities:
+            commorbidities_list.append(_COMORBIDITIES_MAP[key])
+
+    preexistingConditions["values"] = commorbidities_list
+    return preexistingConditions
+
+
+def convert_ethnicity(raw_ethnicity: str):
+    if raw_ethnicity == "Preta":
+        return "Black"
+    elif raw_ethnicity == "Parda":
+        return "Mixed"
+    elif raw_ethnicity == "Amarela":
+        return "Asian"
+    elif raw_ethnicity == "Branca":
+        return "White"
+    elif raw_ethnicity == "Indigena":
+        return "Indigenous"
+
+
+def convert_demographics(gender: str, age: str, ethnicity: str):
+    if not any((gender, age, ethnicity)):
+        return None
+    demo = {}
+    if gender:
+        demo["gender"] = convert_gender(gender)
+    if age:
+        if age == "80 e mais":
+            demo["ageRange"] = {
+                "start": 80,
+                "end": 120
+            }
+        elif age == "<1":
+            demo["ageRange"] = {
+                "start": 0,
+                "end": 0
+            }
+        else:
+            age_range = age.split(' a ')
+            demo["ageRange"] = {
+                "start": float(age_range[0]),
+                "end": float(age_range[1])
+            }
+    if ethnicity != "NAO INFORMADO":
+        demo["ethnicity"] = convert_ethnicity(ethnicity.title())
+    return demo
+
+
+def convert_notes(raw_commorbidities: str, raw_notes_neighbourhood: str, raw_notes_indigenousEthnicity: str):
```

Comment:
Again to be consistent with the Paraiba parser, if Others is present as a comorbidity, add 'Unspecified pre-existing condition' to notes.